samples/cloudwatch-applicationsignals-mcp/get-enablement-guide-samples/docker-apps/nodejs/express/app.js

```diff
@@ -0,0 +1,62 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+const express = require('express');
+const { S3Client, ListBucketsCommand } = require('@aws-sdk/client-s3');
+const logger = require('pino')();
+
+const HOST = process.env.HOST || '0.0.0.0';
+const PORT = parseInt(process.env.PORT || '8080', 10);
+
+const app = express();
+
+const s3Client = new S3Client({ region: 'us-east-1' });
```

Comment:
nit: The region is hardcoded to `us-east-1`. It won't impact functionality. But the sibling Python app respects to AWS_REGION. 
https://github.com/awslabs/mcp/blob/main/samples/cloudwatch-applicationsignals-mcp/get-enablement-guide-samples/docker-apps/python/flask/app.py#L25